package.json

```diff
@@ -11,7 +11,7 @@
     "gulp": "3.8.7",
     "gulp-filelog": "0.4.1",
     "gulp-include": "1.1.1",
-    "gulp-sass": "1.3.3",
+    "gulp-sass": "2.0.4",
```

Comment:
`gulp-sass` 1.3.3 "requires" Node 0.12.5. But I'm running 0.10.35 locally and it's still working. And Node itself is at 4.2.1 now so shrug.